components/PageControl/tests/unit/PageControlExampleTests.m

```diff
@@ -130,4 +130,16 @@ - (void)testCurrentPageGetsUpdatedWhenOffsetIsChanged {
   XCTAssertEqual(pageControl.currentPage, page);
 }
 
+- (void)testResetNumberOfPagesToZero {
```

Comment:
Glad to see a covering test for the change. :) My only request would be that we explicitly check for no exception.  Something like this:

```objectivec
- (void)testResetNumberOfPagesToZero {
  // Given
  MDCPageControl *pageControl = [[MDCPageControl alloc] init];
  pageControl.numberOfPages = 3;
  NSException *exception;

  // When
  @try {
    pageControl.numberOfPages = 0;
    pageControl.currentPage = 0;
  }
  @catch (NSException *e) {
    exception = e;
  }

  // Then
  XCTAssertNil(exception, @"PageControl crashed with exception: %@", exception);
}
```

Reply:
+1

Reply:
Thanks for the review & feedback! I've incorporated the suggested change, which I think is a cleaner way to do it, to the unit test.